config/config_params.go

```diff
@@ -180,6 +180,7 @@ type Config struct {
 	FailsafeOutboundHostPorts []ProtoPort `config:"port-list;udp:53,udp:67,tcp:179,tcp:2379,tcp:2380,tcp:6666,tcp:6667;die-on-fail"`
 
 	KubeNodePortRanges []numorstring.Port `config:"portrange-list;30000:32767"`
+	NatPortRange       numorstring.Port   `config:"portrange;;local"`
```

Comment:
Just spotted the `local` on here; that shouldn't be needed - no reason to limit this config to env vars only

Reply:
Maybe you added this to get the tests to pass without the libcalico-go change?

Reply:
Yes, I added that to make the tests pass. I removed it as requests.